backend/app/exceptions.py

```diff
@@ -282,3 +282,11 @@ class InvalidMatchedRewardsStrategy(OctantException):
 
     def __init__(self):
         super().__init__(self.description, self.code)
+
+
+class DelegationAlreadyExists(OctantException):
+    code = 403
```

Comment:
maybe 400 or 409 will be more appropriate here?

Reply:
changed to 400

Reply:
changed to 400